remote/src/main/scala/org/apache/pekko/remote/transport/PekkoPduCodec.scala

```diff
@@ -128,7 +128,7 @@ private[remote] trait PekkoPduCodec {
  * INTERNAL API
  */
 @nowarn("msg=deprecated")
-private[remote] object PekkoPduProtobufCodec$ extends PekkoPduCodec {
```

Comment:
Hum

Reply:
![image](https://github.com/apache/incubator-pekko/assets/501740/cd474f83-65aa-4903-82bd-f271494d0ec2)

Reply:
Seems IDEA messed this.